app/src/main/java/com/greenfox/kalendaryo/fragments/KalendarFragment.java

```diff
@@ -55,7 +56,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         floatingActionButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
check which way is preferred (@OnClick or onClickListener())

Reply:
Android guide (https://developer.android.com/guide/topics/ui/controls/button) considers the two options equivalent, but there are some comments on the internet which highlight the drawbacks of using android:onClick (eg. hard to see through, less flexible, causes inconsistency when using onLongClickListener()).